site/src/components/pages/hub/block-data-container.tsx

```diff
@@ -364,9 +364,15 @@ export const BlockDataContainer: FunctionComponent<BlockDataContainerProps> = ({
             }}
             onClose={() => setAlertSnackBarOpen(false)}
           >
-            <Alert severity="warning">
-              Please fix the errors in the <b>Data Source</b> before proceeding.{" "}
-            </Alert>
+            <Alert
+              description={
+                <>
+                  Please fix the errors in the <b>Data Source</b> before
```

Comment:
Not related to this PR but can you do a driveby fix by renaming this to "Block Properties"

```suggestion
                  Please fix the errors in <b>Block Properties</b> before
```